pyproject.toml

```diff
@@ -146,6 +146,7 @@ scikit-learn = { version = ">=1.3,<2", python = ">=3.10,<3.13" }
 seaborn = { version = ">=0.12.2,<1", python = ">=3.10,<3.13" }
 leafmap = { version = ">=0.29.6,<1", python = ">=3.10,<3.13" }
 lonboard = { version = ">=0.5.0,<1", python = ">=3.10,<3.13" }
+kafka-python = "^2.0.2"
```

Comment:
Maybe this isn't necessary? Given that the notebook outputs are supposed to be frozen.

Reply:
Yep, this should be removed from `pyproject.toml` as well as `requirements-dev.txt`.

Reply:
Done!